docs/handleiding/index.rst

```diff
@@ -0,0 +1,30 @@
+.. _handleiding_index:
+
+Inrichting van het Burgerportaal
+==============================
+
+Vormgeving en links naar de gemeentelijk website
+-------------------------------------------------
+
+Het GPP-Burgerportaal,  an het `NL Design System (NLDS) <https://nldesignsystem.nl/>`_. Daarnaast zijn er diverse onderdelen van de interface die gemeente-specifiek ingericht kunnen worden. 
+
+Let op: het is belangrijk om na te denken over deze onderdelen vóórdat een gemeente het GPP-Burgerportaal gaat installeren. Zie ook `het stappenplan op de community website <https://www.gpp-woo.nl/implementatie>`_. 
+
+In de `lijst met omgevingsvariabelen <https://github.com/GPP-Woo/GPP-burgerportaal?tab=readme-ov-file#burgerportaal>`_ staat uitgelegd welke gegevens het Burgerportaal nodig heeft van de gemeente, om aan te sluiten bij de gemeentelijke website. Het gaat dan om die variabelen die beginnen met ``RESOURCES:``
+
+Informatie op de homepage
+-----------------------------
+Het is mogelijk om op de homepage van het Burgerportaal gemeentespecifieke informatie te tonen. Deze informatie moet zijn opgemaakt in HTML-format. Hierin is beperkte opmaak mogelijk: kopjes van 2 niveaus, links naar andere websites en genummerde of bullet-lijsten. Dit kan met de volgende HTML-elementen: ``<h1>``, ``<h2>``, ``<p>``, ``<a>``, ``<ul>``, ``<ol>``, ``<li>``.
+
+Video 
```

Comment:
In de lijst met omgevingsvariabelen zeggen we iets over (licht) teruggeschroefde security en het embedden van video's. Denk je dat we dat hier ook moeten noemen?

Reply:
Ja, dat vind ik inderdaad een lastige. Ik dacht: ik link zoveel mogelijk naar lijsten met EnvVars, zodat we ze niet op twee plekken hoeve bij te houden. 
Maar dit is vermoedelijk wel een onderwerp dat we ook hier moeten aanstippen, ja. 
Ik ga het opnemen.